pages/_app.js

```diff
@@ -7,7 +7,7 @@ import { getCardById } from "../lib/data";
 export default function App({ Component, pageProps }) {
   const [id, setId] = useState(undefined);
   useEffect(() => {
-    setId(Math.floor(Math.random() * 77 + 0));
+    setId(Math.floor(Math.random() * 77));
   }, []);
 
   const dailyCard = getCardById(id);
```

Comment:
On the initial render, `id` will be undefined. Not sure what `getByCardId()` will return in this case. You could return `null` or `undefined` in that case to prevent rendering stuff with an undefined `dailyCard` object.